src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1900,7 +1901,8 @@ PeerImp::checkPropose (Job& job,
     }
     else
     {
-        if (app_.getOPs().getConsensusLCL() == peerPos.proposal().prevLedger())
+        if (app_.getOPs().getConsensusLCL() == peerPos.proposal().prevLedger() ||
```

Comment:
Same micro-nit: reversing the check can short-circuit the more expensive `uint256` equality check.